azure-pipelines.yml

```diff
@@ -18,20 +27,19 @@ steps:
     secureFile: 'GoogleServiceAccountKey.json'
```

Comment:
Is this automatically set when we turned on that plugin? Or is this something you added on the DevOps side of things?

Reply:
This is something I had to upload to DevOps, that is a key that allows the pipeline to use the service account

Reply:
https://dev.azure.com/SOC-OIT/CDOT/_library?itemType=SecureFiles&view=SecureFileView&secureFileId=99077f4b-bea2-4fe5-ba85-fd33fce50adb&path=GoogleServiceAccountKey.json

Reply:
I figured it was the service account auth file, but just wasn't sure if this was something we had to do manually or not. Thanks for the clarification!